pom.xml

```diff
@@ -475,6 +479,48 @@
             <argLine>-Xmx1024m -XX:MaxPermSize=128m -Dfile.encoding=UTF-8</argLine>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.jacoco</groupId>
+          <artifactId>jacoco-maven-plugin</artifactId>
+          <version>0.7.5.201505241946</version>
+          <executions>
+            <execution>
+              <id>default-prepare-agent</id>
+              <goals>
+                <goal>prepare-agent</goal>
+              </goals>
+            </execution>
+            <execution>
+              <id>default-report</id>
+              <phase>prepare-package</phase>
+              <goals>
+                <goal>report</goal>
+              </goals>
+            </execution>
+            <execution>
+              <id>default-check</id>
+              <goals>
+                <goal>check</goal>
+              </goals>
+              <configuration>
+                <!-- Do not fail the build if rules are not met -->
+                <haltOnFailure>false</haltOnFailure>
+                <rules>
+                  <rule>
+                    <element>BUNDLE</element>
+                    <limits>
+                      <limit>
+                        <counter>LINE</counter>
+                        <value>COVEREDRATIO</value>
+                        <minimum>0.9</minimum>
```

Comment:
I'll confess, I'm doing TDD most of the time and not even meaning Toni Driven Development, but the real thing ;)

Like Jiri mentioned, 90% will be a problem with the MVP view implementations we have. We only test the presenter since, the view is GWT and unit testing it is hard, but we also keep the views as simple as possible ( = just setting values or styles ). The view part is still Java code, not sure if it will be possible to ignore *ViewImpl.java files when counting the %?